<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingComDialog</class>
 <widget class="QDialog" name="SettingComDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config Com</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Icons/settings.png</normaloff>:/Icons/settings.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="SetgroupBox_Cal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings Calibrator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="ParComgroupBox_Cal">
          <property name="title">
           <string>Parameter COM</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="BaudRateLabel_Cal">
               <property name="text">
                <string>BaudRate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="BoudrateBox_Cal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="DataBitsLabel_Cal">
               <property name="text">
                <string>DataBits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="DataBitsBox_Cal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="ParityLabel_Cal">
               <property name="text">
                <string>Parity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ParityBox_Cal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="StopBitsLabel_Cal">
               <property name="text">
                <string>StopBits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="StopBitsBox_Cal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="FlowControlLabel_Cal">
               <property name="text">
                <string>FlowControl</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="FlowControlBox_Cal"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_SelCom_Cal">
          <property name="title">
           <string>Select COM</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="COMLabel_Cal">
               <property name="text">
                <string>COM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="SetComBox_Cal"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="ApplyButton_Cal">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CleanButton_Cal">
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="SetgroupBox_Volt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings Voltmeter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="ParComVoltgroupBox_Volt">
          <property name="title">
           <string>Parameter COM</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="BaudRateLabel_Volt">
               <property name="text">
                <string>BaudRate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="BoudrateBox_Volt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="DataBitsLabel_Volt">
               <property name="text">
                <string>DataBits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="DataBitsBox_Volt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="ParityLabel_Volt">
               <property name="text">
                <string>Parity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ParityBox_Volt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="StopBitsLabel_Volt">
               <property name="text">
                <string>StopBits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="StopBitsBox_Volt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="FlowControlLabel_Volt">
               <property name="text">
                <string>FlowControl</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="FlowControlBox_Volt"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_SelCom_Volt">
          <property name="title">
           <string>Select COM</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="COMLabel_Volt">
               <property name="text">
                <string>COM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="SetComBox_Volt"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QPushButton" name="ApplyVoltButton_Volt">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CleanVoltButton_Volt">
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
